mdbenchmark/generate.py

```diff
@@ -124,33 +89,39 @@ def generate(name, gpu, module, host, min_nodes, max_nodes, time, list_hosts):
             'Could not find template for host \'{}\'.'.format(host),
             param_hint='"--host"')
 
-    if not module:
-        raise click.BadParameter(
-            'You did not specify which gromacs module to use for scaling.',
-            param_hint='"-m" / "--module"')
-
-    # Provide some output for the user
-    number_of_benchmarks = (len(module) * (max_nodes + 1 - min_nodes))
-    run_time_each = '{} minutes'.format(time)
-    console.info(
-        'Creating a total of {} benchmarks, with a run time of {} each.',
-        number_of_benchmarks, run_time_each)
-
     for m in module:
-        directory = '{}_{}'.format(host, m)
-        gpu_string = '.'
+        # Here we detect the mdengine (GROMACS or NAMD).
+        engine = detect_md_engine(m)
+        # Note: NAMD input files can have relative imports. To avoid issues
+        # when copying the files, we currently only allow the use of absolute
+        # paths.
+        if engine == namd:
+            console.warn('ATTENTION NAMD USERS!'
```

Comment:
We probably can skip the first line (`ATTENTION` is not needed when the message is prefixed with a bold and colored `WARNING`).  Use something like this instead: console.warn('NAMD support is experimental. All input files must be in the current directory. Parameter paths must be absolute. Only crude file checks are performed!')